### DISCUSSION TITLE
Log shadowsocks

### DISCUSSION THREAD

**Discussion by Paulpatou** [qdm12/gluetun#693] [Ideas]
Hello,
I have a question about the log provided by shadowsocks in Gluetun.
I use shadowsocks in another project but I can't get logs as clean as what Gluetun provides.
=> ` 2021/10/28 07:39:36 INFO shadowsocks: UDP proxying 172.16.170.13:34847 to 51.255.209.72:123`

I tried with the verbose mode of shadowsocks but there is too much information that I don't need to get.

Is it possible to have an explanation on how you do it?

**Comment by qdm12** [comment]
Hey there!

I wrote my own implementation 😄 It's at https://github.com/qdm12/ss-server
It does respect the protocol (otherwise, well, it would just not work), but I really did what I wanted here and there, so it will differ from other existing implementations.
If you want feel free to use the container image `qmcgaw/ss-server` for a SS server, it should work decently well. It's also relatively stable since I barely change the repository these days.
